sp800-63a/sec6_dc.md

```diff
@@ -2,64 +2,22 @@
 
 <div class="breaker"></div>
 
-# 6. Leveraging Antecedent Proofing Events
+# <a name="resuse_proof"></a> 6. Derived Authenticators
 
 _This section is normative._
 
-Possession of an authenticator, bound to an identity proofed at a given IAL, should allow a claimant to obtain additional authenticators, per CSP policy.  This section details requirements for two specific use cases:
+It is beneficial to an individual and an organization to re-proof as little as possible.  Wherever possible, verifying that an individual has already been successfully identity proofed SHOULD be performed.
 
-1. A _claimant_ obtains a secondary, or derived authenticator, for use only within the limits and authorizations of the primary authenticator.  For example, this could be a derived PIV credential in federal use cases, or a temporary authenticator. See [Section 6.1](#dc) for more details.
-2. An _applicant_ seeks to obtain another primary authenticator from a CSP that did not issue the original authenticator. For example, if an applicant wants to switch CSPs and/or have another authenticator at a new CSP for other uses (e.g. basic browsing vs. financial). See [Section 6.2](#prior) for more details.
+>Note: This guideline supports using digital credentials, in addition to physical credentials, as identity evidence.  See [Section 5.2.](#validate) for more information.
 
-## <a name="dc"></a> 6.1. Issuance Requirements for Derived Authenticators
+There are two specific use cases to consider when leveraging the proofing associated with an individual:
 
-This section applies when the secondary authenticator is directly tied to the authorizations of having, and proving possession of, a primary authenticator, typically issued by the same CSP.
+1. A _claimant_ obtains a secondary, or derived authenticator, for use only within the limits and authorizations of the primary authenticator.  For example, this could be a derived PIV credential in federal use cases, or a temporary authenticator. See [Section 6.1](#dc) for more details. *This section covers this use case.*
```

Comment:
I see that some places we refer to a "primary authenticator" and in others an "original authenticator". I prefer the latter; "primary" implies some supremacy of one authenticator over others when they're really on equal footing. For example, if the original authenticator is revoked (perhaps because of loss), the derived authenticator still is usable.